bbj-vscode/test/validation.test.ts

```diff
@@ -351,10 +351,7 @@ describe('BBj validation', async () => {
         expectNoIssues(validationResult);
     });
 
-    /**
-     * Waiting for answer in #149
-     */
-    test.skip('IF as last child of a compound statement', async () => {
+    test('IF as last child of a compound statement', async () => {
         const validationResult = await validate(`
         debug = 1
```

Comment:
But why does it work when I add a line break? The FI or ENDIF is still missing.
```
debug = 1
PRINT "Foo";
if debug then
PRINT "top_rs_row=",3
```

Reply:
Thanks for testing!
This is now a multi line IF, not part of #149.
We need to check if missing FI is a valid case or not and open a new issue.